sycl/source/detail/queue_impl.hpp

```diff
@@ -264,7 +264,17 @@ class queue_impl {
       destructorNotification();
 #endif
       throw_asynchronous();
-      getAdapter()->call<UrApiKind::urQueueRelease>(MQueues[0]);
+      auto status =
+          getAdapter()->call_nocheck<UrApiKind::urQueueRelease>(MQueues[0]);
+      // If loader is already closed, it'll return a not-initialized status
+      // which the UR should convert to SUCCESS code. But that isn't always
+      // working on Windows. This is a temporary workaround until that is fixed.
+      // TODO: Remove this workaround when UR is fixed, and restore
```

Comment:
could we please create a JIRA task for this todo (blocked by UR issue)?
we have a lot of TODOs in code not tracked and forgotten.